selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsTest.java

```diff
@@ -0,0 +1,263 @@
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.dashboard.CreateWorkspace;
+import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
+import org.eclipse.che.selenium.pageobject.dashboard.DashboardProject;
+import org.eclipse.che.selenium.pageobject.dashboard.DashboardProject.Template;
+import org.eclipse.che.selenium.pageobject.dashboard.DashboardWorkspace;
+import org.eclipse.che.selenium.pageobject.dashboard.DashboardWorkspace.TabNames;
+import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Skoryk Serhii */
+public class WorkspaceDetailsTest {
+  private static final String WORKSPACE = NameGenerator.generate("java-mysql", 4);
+  private static final String PROJECT_NAME = "web-java-petclinic";
+  private List<String> AgentsList =
+      Arrays.asList(
+          "C# language server",
+          "Exec",
+          "File sync",
+          "Git credentials",
+          "JSON language server",
+          "PHP language server",
+          "Python language server",
+          "SSH",
+          "Terminal",
+          "TypeScript language server",
+          "Workspace API");
+  private List<Boolean> AgentsStateList =
+      Arrays.asList(false, true, false, false, false, false, false, true, true, false, true);
+  private List<String> VariablesList =
+      Arrays.asList("MYSQL_DATABASE", "MYSQL_PASSWORD", "MYSQL_ROOT_PASSWORD", "MYSQL_USER");
+  private List<String> ValuesList = Arrays.asList("petclinic", "password", "password", "petclinic");
+
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private Loader loader;
+  @Inject private NavigationBar navigationBar;
+  @Inject private CreateWorkspace createWorkspace;
+  @Inject private Dashboard dashboard;
+  @Inject private DashboardWorkspace dashboardWorkspace;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+  @Inject private NotificationsPopupPanel notificationsPopupPanel;
+  @Inject private DashboardProject dashboardProject;
+  @Inject private Consoles consoles;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    dashboard.open();
+    createWsFromJavaMySqlStack();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(WORKSPACE, defaultTestUser.getName());
+  }
+
+  @Test(priority = 0)
+  public void workingWithEnvVariables() {
+    dashboardWorkspace.selectTabInWorspaceMenu(TabNames.ENV_VARIABLES);
+    dashboardWorkspace.selectMachine("Environment variables", "dev-machine");
+    dashboardWorkspace.clickOnAddEnvVariableButton();
+    dashboardWorkspace.checkAddNewEnvVarialbleDialogIsOpen();
+    dashboardWorkspace.addNewEnvironmentVariable("logi", "qaadmin");
```

Comment:
**qaadmin** is the name which we are using on staging. Please, avoid usage of it in tests directly.